lib/logstash/codecs/json.rb

```diff
@@ -37,7 +37,15 @@ def register
   def decode(data)
     data = @converter.convert(data)
     begin
-      yield LogStash::Event.new(LogStash::Json.load(data))
+      decoded = LogStash::Json.load(data)
+      if decoded.is_a?(Array)
+        decoded.each {|item|
```

Comment:
haha, no worries. You should see my ruby after I write C or Javascript. SEMICOLONS EVERYWHERE.